HoloRepositoryDemoApplication/Assets/HoloStorageConnector/Scripts/HologramInstantiationSettings.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Microsoft.MixedReality.Toolkit.UI;
+using UnityEngine.SceneManagement;
+using UnityEngine;
+using System;
+
+namespace HoloStorageConnector
+{
+    public class HologramInstantiationSettings
+    {
+        #region Properties
+        /// <summary>
+        /// Set a name for the loaded model
+        /// </summary>
+        public string Name { get; set; } = "LoadedModel";
+        /// <summary>
+        /// Set position for the loaded model, the parameter should be a Vector3 value
+        /// </summary>
+        public Vector3 Position { get; set; } = new Vector3(0f, 0f, 0f);
+        /// <summary>
+        /// Set rotation for the loaded model, the parameter should be a Vector3 value
+        /// </summary>
+        public Vector3 Rotation { get; set; } = new Vector3(0f, 0f, 0f);
+        /// <summary>
+        /// Real size in the scene, The longest side of the loaded object will be set to this value
+        /// </summary>
+        public float Size { get; set; } = 1f;
```

Comment:
Maybe 0.1 would be a better default value, given how huge the default value was in your demonstration on Friday :)